test/integration/ios.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 - hosts: ios
   gather_facts: no
-  connection: local
+  connection: network_cli
```

Comment:
This should maybe go into ansible-test as we will not be directly invoking `ios.yaml`, we'd instead be invoking each module